_pages/en/it-picture-long-term.md

```diff
@@ -38,59 +45,47 @@ They can also use the digital services on behalf of clients.
 GC departments have common kiosks to offer services, such as passports, taxes, grants as well as benefits from ESDC.
 Outreach programs are deployed to raise awareness in low uptake areas.
 
-### IoT Integration
-
-ESDC does not develop products or services that compete with the private sector.
-IoT devices will begin to permeate all aspects of life and ESDC should integrate itself into the ecosystem. 
-
-An example of the repercussions of such an implementation can be found [here](https://github.com/sara-sabr/ITStrategy/blob/master/wip-tec/en/human-development-life-cycle.md).
-
-In a service-based environment, success is not measured by the number of users your service attracts, but rather the number of users amassed by the services that leverage your service. 
-- TODO: Add source (Microsoft)
+## Private Sector
 
-User privacy must still be preserved and modern digital identity recognition technologies are leveraged (e.g. facial, biometrics).
+### Any platform, any device, 3rd party integration
 
-For more details on its implementation, please refer to the [Business section](##Businesses)
+Any platform or device can be integrated with GC and ESDC services and development is often done in partnership with the private sector.
```

Comment:
Any platform or device ... Wouldn't we mean smart internet accessible devices?

Often? How often? Is this an increasing trend? What would be the evidence for that?

Reply:
What are smart internet accessible devices?

It's using the OneGC vision

Reply:
Hmm ... if that's the language they use, then all is well. Can we link to some of their documentation to bolster the point?